x/crosschain/keeper/cctx_gateway_zevm.go

```diff
@@ -28,7 +28,7 @@ func (c CCTXGatewayZEVM) InitiateOutbound(
 
 	if err != nil && !isContractReverted {
 		// exceptional case; internal error; should abort CCTX
-		config.CCTX.SetAbort(err.Error())
+		config.CCTX.SetAbort("Aborted while trying to handle EVM deposit", err.Error())
```

Comment:
Related topmy other commend , if we declare constants , we can use that in this error message also; 
If using the terminology you already have, this would be 

config.CCTX.SetAbort("Aborted while trying to handle EVM deposit", fmt.sprintf("Deposit Error :%s",err.Error))